### PR TITLE
Documented impl From on line 367 of libserialize/json.rs

### PR DESCRIPTION
**Pull request by Ajacmac**
This is for the impl From mentioned in #51430 assigned to @skade .

Hopefully I didn't miss anything/get anything wrong. I looked over another PR for another part of this same issue to see what the proper formatting was, etc.

Thanks!

### CODE REVIEW

**Comment by Ajacmac** [comment]
Hmmm. So the name on the PR is wrong. Different user, same computer. Is there any way to change that now?

**Comment by Aaronepower** [comment]
Triage: Assigning this to @frewsxcv

**Comment by Aaronepower** [comment]
Triage: r? @frewsxcv

**Comment by TimNN** [comment]
Ping from triage @frewsxcv / @rust-lang/docs: This PR requires your review.

File: src/libserialize/json.rs [hunk 1]
```diff
@@ -365,6 +365,9 @@ impl std::error::Error for EncoderError {
 }
 
 impl From<fmt::Error> for EncoderError {
+    /// Converts a `fmt::Error` into `EncoderError`
```
Comment (GuillaumeGomez) [inline src/libserialize/json.rs]: Please change to "Converts a [\`fmt::Error\`] into \`EncoderError\`."

**Comment by Ajacmac** [comment]
My bad. It should be fixed now.

**Comment by GuillaumeGomez** [comment]
Thanks!

@bors: r+ rollup

**Pull request merged**